.github/workflows/main.yml

```
@@ -0,0 +1,30 @@
+name: build
+
+on: [push, pull_request, workflow_dispatch]
```

lets make this the same as our ejabberd workflow. It seems like right now we have each version of otp run twice
```suggestion
on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
```